Formula/synergy-core.rb

```diff
@@ -0,0 +1,109 @@
+class SynergyCore < Formula
+  desc "Synergy, the keyboard and mouse sharing tool"
+  homepage "https://symless.com/synergy"
+  url "https://github.com/symless/synergy-core/archive/refs/tags/1.14.3.5-stable.tar.gz"
+  sha256 "9259fc8aa02610121932cd5887c4d878beb08b2f7190e834c0584d53b36f5f9b"
+
+  # The synergy-core/LICENSE file contains the following preamble:
+  #    This program is released under the GPL with the additional exemption
+  #    that compiling, linking, and/or using OpenSSL is allowed.
+  # This preamble is followed by the text of the GPL-2.0.
+  #
+  # The SPDX license exception list does not contain a symbol corresponding
+  # to synergy-core's OpenSSL exception. The openvpn-openssl-exception
+  # exception symbol is very similar but it is not exactly the same.
+  license "GPL-2.0-only"
```

Comment:
It can't be "GPL-2.0-only` and have an exception for OpenSSL, it's "GPL 2.0 with exception" (which means that there's absolutely no bare GPL code in synergy), or it's "GPL 2.0" in which case you can't add an exception to have it link against a library licensed under a non-GPL compatible license.

Reply:
I understand that Synergy is "GPL 2.0 with exception" and not "GPL-2.0-only". No one is saying it is "GPL-2.0-only". The `license` method in `brew` doesn't provide a way to communicate "GPL 2.0 with Synergy's exception", which is why I put a comment above the line to explain that it's not "GPL-2.0-only".